### PR TITLE
Make project VS Code Dev Container compatiable

### PR DESCRIPTION
**Pull request by talk2MeGooseman**
Also added in solargraph for better VS Code integration with Rails

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,15 @@
 * `docker-compose run web bundle exec rails db:create db:setup`
 * `docker-compose up`
 
+**Setup Using Visual Studio Code**
+
+1. Open project directory in VS Code
+1. Press F1, and select `Remote-Containers: Reopen in Container...`
+1. Wait a few minutes as it pulls image down and builds Dev Conatiner Docker image (this should only need to happen once unless you modify the Dockerfile)
```
Comment (thatonejason) [inline README.md:13]: Conatiner => Container

File: README.md [hunk 2]
```diff
@@ -6,6 +6,15 @@
 * `docker-compose run web bundle exec rails db:create db:setup`
 * `docker-compose up`
 
+**Setup Using Visual Studio Code**
```
Comment (thatonejason) [inline README.md:9]: If this is optional, I'd like to label it that way, especially if it only works if you have access to the service. If it totally works as a local solution, then that can be made more clear, too. But if the project still works normally using just Docker, I wouldn't want a candidate to feel like they have to do this part. I just don't know how common knowledge this is so people would be able to decide for themselves whether they should attempt this or not.

**Comment by thatonejason** [comment]
I'm not very familiar with the dev container thing but, with these changes, does the Docker setup still work normally as well?

File: README.md (hunk 2 above)
Comment (talk2MeGooseman) [reply README.md:9]: Luckily doing these steps only requires Docker and VS Code. The feature is already built-in and using the existing container development workflow to make it even easier. But identifying this is an optional route is a good call

**Review by thatonejason — APPROVED** [review]

**Pull request merged**